### DISCUSSION TITLE
Is there any resolution limitations?

### DISCUSSION THREAD

**Discussion by luckkmaxx** [lamarios/clipious#327] [Q&A] [answered]
Hi there. Im not sure if this is a bug or a known limitation.
My app only plays up to 720p. No option to set higher resolutions.
Im using my own invidious instance, where I can play fhd, 4k videos from the website. But same videos only can be played up to 720p in clipious.

**Comment by lamarios** [comment, marked as answer]
You might need to switch DASH either in the video options or the app settings under the video player category.

**Comment by luckkmaxx** [comment]
Thanks! I can see now any avaliable resolution. But playback isn't working. The videos keeps at 0:00, and if I go forward then appears a [ ! ] icon in the middle.
It works in official instances.
Have I missed something related to dash in my instance setup?

**Comment by lamarios** [comment]
Try enable or disable the proxy video setting as well based on current set up.

**Comment by luckkmaxx** [comment]
> Try enable or disable the proxy video setting as well based on current set up.

Nothing change activating proxy. I will take a look to my instance setup.

**Comment by luckkmaxx** [comment]
Ohhh I suspect whats going on. Need to test. My instance has an internal domain name, wich cant be reached from outside lan.

**Comment by luckkmaxx** [comment]
YES! the instance need to be reached from WAN to get DASH working.
